packages/react-search-ui/src/SearchProvider.js

```diff
@@ -25,6 +27,11 @@ class SearchProvider extends Component {
 
   componentDidMount() {
     const { config } = this.props;
+    config.a11yNotificationMessages = {
```

Comment:
I think mutating props could work in this case, but I find it hard to think where it's ok to mutate and where it's not, so I try to never mutate them. I would create a new variable here:
```js
const updatedConfig = {
   ...config,
   a11yNotificationMessages: {
      ...defaultA11yMessages,
      ...config.a11yNotificationMessages
   }
}
```
What do you think?

Reply:
Oh good catch @yakhinvadim. I would definitely recommend not mutating config.

Reply:
Addressed in 6dced84!